templates/base/client/.rtag/types.ts.hbs

```diff
@@ -5,6 +5,22 @@ export enum {{@key}} {
   {{label}},
   {{/each}}
 }
+{{else if (eq type "union")}}
+{{#each options}}
+interface {{@../key}}_{{> renderArg}} {
+  type: "{{> renderArg}}"
+  value: {{> renderArg}}
+}
+{{/each}}
+export type {{@key}} =
+{{#each options}}
+ | {{@../key}}_{{> renderArg}}
```

Comment:
can't we just do `| {{> renderArg}}` directly?

Reply:
no because we need to put the type string literal on there

Reply:
per prior comment maybe there's a better way